modules/apps/portal-search/portal-search-rest-impl/src/main/java/com/liferay/portal/search/rest/internal/odata/entity/v1_0/SearchResultEntityModel.java

```diff
@@ -45,12 +45,14 @@ public SearchResultEntityModel() {
 				"description",
 				locale -> Field.getSortableFieldName(
 					LocalizationUtil.getLocalizedName(
-						Field.DESCRIPTION, LocaleUtil.toLanguageId(locale)))),
+						Field.DESCRIPTION, LocaleUtil.toLanguageId(locale))),
+				locale -> Field.DESCRIPTION),
 			new StringEntityField(
 				"title",
 				locale -> Field.getSortableFieldName(
 					LocalizationUtil.getLocalizedName(
-						"localized_title", LocaleUtil.toLanguageId(locale)))));
+						"localized_title", LocaleUtil.toLanguageId(locale))),
```

Comment:
I didn't test this so it might work just fine, but it seems that headless teams pattern to changes per LPS-157425 is slightly different: https://github.com/liferay/liferay-portal/commit/f51bf2d8296acc9de1631b96215e9ace9fb9d620

Would that work?

@AlmirFe when testing can you please validate it that Brian Lee's example here works, too https://liferay.atlassian.net/browse/LPS-157425. Thx!